lib/create-space-api.js

```diff
@@ -43,6 +43,8 @@ import entities from './entities'
  * @prop {function} createRoleWithId
  * @prop {function} getSpaceUser
  * @prop {function} getSpaceUsers
+ * @prop {function} getSpaceMember
```

Comment:
Let's also add this to the typings pls :)

Reply:
Oh definitely! Thanks for reminding me.